x-pack/plugins/rollup/index.js

```diff
@@ -20,7 +21,7 @@ export function rollup(kibana) {
   return new kibana.Plugin({
     id: PLUGIN.ID,
     publicDir: resolve(__dirname, 'public'),
-    require: ['kibana', 'elasticsearch', 'xpack_main'],
+    require: ['kibana', 'elasticsearch', 'xpack_main', 'metrics'],
```

Comment:
I am a bit worried about that. @epixa does that mean, that the `rollup` plugin won't startup as long as any of the required plugins is disabled via the `kibana.yml`?

Reply:
I'm not 100% on the behavior for the legacy plugin `require`, but I'm pretty sure all of Kibana will fail to start up if there's a plugin enabled that `require`s a plugin that is disabled. We don't do any automatic disabling in the legacy world though.

Reply:
@alexwizp Could you please try out what happens if you put `metrics.enabled: false` into your `kibana.yml`, it should not include TSVB then, but will it still come up?

Reply:
Hi @timroes. In that case the server will crash:
![image](https://user-images.githubusercontent.com/31325372/51736303-c4c7ed80-209a-11e9-8578-a94c127fdb82.png)

Reply:
@timroes  Thanks! it's not working! We'll fix

Reply:
resolved